scripts/run-live-build.sh

```diff
@@ -123,7 +123,7 @@ fi
 # user (e.g. other software); this is most useful when multiple variants
 # are built via a single call to "make" (e.g. using the "all" target).
 #
-for ext in ova qcow2 upgrade.tar.gz vhdx vmdk; do
+for ext in ova qcow2 upgrade.tar.gz tar.gz vhdx vmdk; do
```

Comment:
Can we use the extension “.gcp.tar.gz” (or equivalent) to better distinguish it from the upgrade image?

Reply:
I will give this a shot and do a follow-on review if it works.